src/odemis/acq/test/path_test.py

```diff
@@ -476,6 +477,24 @@ def test_wrong_mode(self):
         with self.assertRaises(ValueError):
             self.optmngr.setPath("ErrorMode").result()
 
+    def test_queue(self):
+        """
+        Test changing path multiple times without waiting for it to be complete
+        """
+        tstart = time.time()
+        self.optmngr.setPath("cli")
+
+        # All these ones should get discarded
+        for i in range(5):
+            self.optmngr.setPath("ar")
+            self.optmngr.setPath("mirror-align")
+            self.optmngr.setPath("cli")
```

Comment:
These ones are not started as you do not call `.result()` and therefore only put on the queue?

Reply:
Yes. Because there is already a setPath() going on, and that we immediately ask for more, they end up being queued (and now, immediately discarded).